samples/tab-user-profile/ts/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-user-profile-ts",
+    "source": "officeDev",
+    "title": "Starter solution: Personal tab in TypeScript",
+    "shortDescription": "Create React App personal tab in TypeScript",
```

Comment:
Get user profile information in Outlook using personal tab.

Reply:
Fixed